### ISSUE TITLE
balena scan: Class constructor PinejsClientCore cannot be invoked without 'new'

### ISSUE THREAD

**Issue by pdcastro** [balena-io/balena-cli#1899]
Issue affecting CLI releases 12.3.0 to 12.3.3:
```
sudo balena scan
Class constructor PinejsClientCore cannot be invoked without 'new'
```

A fix will be available shortly.

**Comment by pdcastro** [comment]
This issue is fixed in CLI release v12.3.4 (PR #1900).

@EUdds, @sandersteenhoudt, sorry for the trouble this has caused you and thank you for your patience!

**Comment by EUdds** [comment]
> 
> 
> This issue is fixed in CLI release v12.3.4 (PR #1900).
> 
> @EUdds, @sandersteenhoudt, sorry for the trouble this has caused you and thank you for your patience!

Thanks so much for the quick fix!!
